docs/assets/includes/quality-metrics-resources.md

```diff
@@ -0,0 +1,5 @@
+!!! note
```

Comment:
As discussed yesterday, it now seems excessive to use this note to include a couple of links right before the **See also** section. It's better to just include the **See also** and list all relevant links there. :+1:

Reply:
Done! ✅